skbio/diversity/_driver.py

```diff
@@ -10,7 +10,7 @@
 import itertools
 
 import numpy as np
-import scipy.spatial.distance
+import sklearn.metrics
```

Comment:
add sklearn added to documentation (and ci/xxx requirement files) as a dependency?

Reply:
it is already in ci, adding to docs